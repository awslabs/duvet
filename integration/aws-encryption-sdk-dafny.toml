name = "aws-encryption-sdk-dafny"
repo = "https://github.com/aws/aws-encryption-sdk-dafny"
version = "v4.1.0"
cmd = [
    "make duvet"
]
html_report = "specification_compliance_report.html"
