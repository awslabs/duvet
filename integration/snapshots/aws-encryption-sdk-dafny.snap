version https://git-lfs.github.com/spec/v1
oid sha256:a11629057c9843b44a4af6feedd5144655197167c847e40b8cc0860f96bc99ce
size 1240152
